express.info

```diff
@@ -204,6 +204,8 @@ dependencies[] = context_sitewide_public
 dependencies[] = field_reference_delete
 dependencies[] = express_block_designer
 dependencies[] = express_block_designer_menu
+dependencies[] = erus_lite
+dependencies[] = update
```

Comment:
I don't think having the update module as a dependency makes sense. Disabling that module for whatever reason would also disable the profile. We've done that before. I think the update module should be treated like Varnish, Memcache or other optional modules.